### DISCUSSION TITLE
【2023.11.9】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#1372] [meeting]
#### Meeting replay: [Bilibili](https://m.bilibili.com/video/BV1ag4y1978g?buvid=XU88D268E94E378FF841A457E9BD40787A2C4&from_spmid=main.space.0.0&is_story_h5=false&mid=VoT7ohbgV488Dgj64ONwQw%3D%3D&p=1&plat_id=114&share_from=ugc&share_medium=android&share_plat=android&share_session_id=a050e285-ac65-4a55-8760-aa708226d68b&share_source=WEIXIN&share_tag=s_i&spmid=united.player-video-detail.0.0&timestamp=1699629935&unique_k=3WvZOiE&up_id=1233089591&code=061cnIkl2zW2mc4p8Lnl2ggWd32cnIkt&state=&share_times=1)

**Conference link：**
> **Note**: Every two weeks at 9:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

**Released V3.4, along with the issues it addresses and new features**

**Outline of the meeting:**
- [User case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md) and [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by plum330** [comment]
func (cli *K8sDR) GetConns(ctx context.Context, serviceName string, opts ...grpc.DialOption) ([]*grpc.ClientConn, error) {

	conn, err := grpc.DialContext(ctx, serviceName, append(cli.options, opts...)...)
	return []*grpc.ClientConn{conn}, err
}
When using this method to push message to the user, how to ensure that the user corresponding to the correct connection to be geted.Because I think grpc dial svc name cann't load to the connection where the specified user resides.

**Comment by plum330** [comment]
How to ensure data consistency?Such as adding black list.
// Create 增加黑名单.
func (b *blackDatabase) Create(ctx context.Context, blacks []*relation.BlackModel) (err error) {
	if err := b.black.Create(ctx, blacks); err != nil {
		return err
	}
	return b.deleteBlackIDsCache(ctx, blacks)
}
